ultravox/data/datasets.py

```diff
@@ -821,6 +827,7 @@ def _get_sample(self, row) -> VoiceSample:
         return self._get_transcribe_sample(row, tcol="sentence")
 
 
+# TODO: this dataset can be replaced with GenericVoiceDataset and will be removed/updated in the future.
```

Comment:
Nit: I don't think these TODO comments are entirely correct. For example the random `PROMPTS` and `CODE_TO_LANG` are not replaceable with the current version.

Reply:
These random prompts will still be used, but the corresponding datasets could be specified using the logic following the (updated) GenericVoiceDataset class, so that we have a unified way of specifying datasets for training, validation, and evaluation.